lib/siwapp/commons.ex

```diff
@@ -178,9 +177,21 @@ defmodule Siwapp.Commons do
     Series.changeset(series, attrs)
   end
 
-  @spec maybe_make_default_series(%{optional(any()) => any()}) :: %{optional(any()) => any()}
-  defp maybe_make_default_series(attrs) do
-    if get_default_series(), do: attrs, else: Map.put(attrs, "default", true)
+  @spec maybe_set_new_default_series(%Series{}) :: {:ok, %Series{}}
+  defp maybe_set_new_default_series(series) do
+    first =
+      list_series()
+      |> List.delete(series)
+      |> List.first()
+
+    if first != nil and get_default_series() == series do
+      set_default_series(first.id)
+    end
+  end
+
+  @spec make_default_if_first(%{optional(any()) => any()}, list()) :: %{optional(any()) => any()}
+  defp make_default_if_first(attrs, list) do
+    if list == [], do: Map.put(attrs, "default", true), else: attrs
```

Comment:
we have to talk about all this stuff, I think we can do better. Anyway, I merge this.